app/views/root/transaction.html.erb

```diff
@@ -27,11 +27,15 @@
     <%- end -%>
 
   </section>
-  <%# Enable cross-domain analytics for IER (register to vote) only - not generalised yet %>
-  <% if @publication.slug == 'register-to-vote' %>
-    <%= render :partial => 'transaction_cross_domain_analytics', :locals => { :ga_account_code => 'UA-23066786-5' } %>
-  <%# Enable cross-domain analytics for Civil claims - still not worth generalising yet %>
-  <% elsif @publication.slug == 'accelerated-possession-eviction' %>
-    <%= render :partial => 'transaction_cross_domain_analytics', :locals => { :ga_account_code => 'UA-37377084-12' } %>
-  <% end %>
+  <% #Enable cross-domain analytics for specific start pages based on slug
+    {
+      'register-to-vote' => 'UA-23066786-5',
+      'accelerated-possession-eviction' => 'UA-37377084-12',
+      'renewtaxcredits' => 'UA-43414424-1', # HMRC, https://www.pivotaltracker.com/n/projects/1261204
+    }.each do |slug, account_code|
+      if @publication.slug == slug %>
```

Comment:
This is starting to turn into quite a lot of logic in this view. Is it worth moving some of this logic into either the presenter or a helper? So the view could just call something like `if @publication.ga_account_code` or `if account_code = ga_account_code_for_publication(@publication)`.